ocaml/xapi/message_forwarding.ml

```diff
@@ -4379,5 +4379,24 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
              (fun () ->
                 do_op_on ~__context ~local_fn ~host
                   (fun session_id rpc -> Client.Cluster_host.disable rpc session_id self)))
+
+    let forget ~__context ~self =
+      info "Cluster_host.forget cluster_host:%s" (Ref.string_of self);
+      let cluster = Db.Cluster_host.get_cluster ~__context ~self in
+      let local_fn = Local.Cluster_host.disable ~self in
+      (* We need to ask another host that has a cluster host to mark it as dead.
+       * We might've run force destroy and this host would no longer have a cluster host
+       * *)
+      let other_hosts =
+        Db.Cluster.get_cluster_hosts ~__context ~self:cluster
+        |> List.filter ((<>) self) in
+      match other_hosts with
+      | [] -> info "No other cluster hosts, nothing to do" (* go ahead and finish Host.destroy *)
+      | other_cluster_host :: _ ->
+        let host = Db.Cluster_host.get_host ~__context ~self:other_cluster_host in
+        Xapi_cluster_helpers.with_cluster_operation ~__context ~self:cluster ~doc:"Cluster.remove" ~op:`remove
```

Comment:
Should we update the `doc` and the `op` here?

Reply:
It performs the same operation as remove, it kicks the member out of the cluster.